asset_dashboard/views.py

```diff
@@ -349,13 +364,18 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
         phase = Phase.objects.get(id=self.kwargs['pk'])
+        phase_list = Phase.objects.filter(project=phase.project).values(
+            'id', 'phase_type', 'estimated_bid_quarter', 'status', 'year'
+        )
 
         context.update({
             'phase': phase,
-            'project': Project.objects.filter(phases=phase)[0],
+            'project': phase.project,
             'props': {
                 'phase_id': phase.id,
-                'phase_name': phase.name
+                'phase_name': phase.name,
+                'phases': json.dumps(list(phase_list), cls=DjangoJSONEncoder),
+                'is_countywide': phase.project.countywide
```

Comment:
need all this in context so i can use the data in the react component. i need these pieces of data to hydrate the countywide toggle and have a list of phases for the dropdown menu